middleware/rate-limiter.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+const {
+    RateLimiterMySQL,
+    RateLimiterMemory
+} = require('rate-limiter-flexible');
+const { startsWith } = require('lodash');
+
+const { sequelize } = require('../db/models/index');
+const appData = require('../common/appData');
+const logger = require('../common/logger').child({
+    service: 'rate-limiter'
+});
+
+const sqlDialect = startsWith(process.env.DB_CONNECTION_URI, 'sqlite://')
+    ? 'sqlite'
+    : 'mysql';
+
+const rateLimiterConfigs = {
+    failByUsername: {
+        maxPoints: 5,
```

Comment:
Is this the number of attempts the user has? If so can we set this to 10 which is the minimum value recommended in https://design-system.service.gov.uk/patterns/passwords/